src/plugins/intel_cpu/tests/unit/shape_inference_test/string_tensor_pack_shape_inference_test.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include "common_test_utils/test_assertions.hpp"
+#include "utils.hpp"
+#include "string_tensor_pack_shape_inference.hpp"
+
+using namespace ov;
+using namespace ov::intel_cpu;
+using ov::op::v0::Constant;
+using ov::op::v0::Parameter;
+
+class StringTensorPackStaticTestSuite : public ::testing::TestWithParam<std::tuple<
+                                                              Shape,                        // begins/ends indices shape
+                                                              std::vector<size_t>,          // begins
+                                                              std::vector<size_t>,          // ends
+                                                              std::vector<uint8_t>          // symbols
+                                                              >> {};
+
+TEST_P(StringTensorPackStaticTestSuite, StringTensorPackStaticShapeInference) {
+    const auto& param = GetParam();
+    const auto& indices_shape = std::get<0>(param);
+    const auto& begins_param = std::get<1>(param);
+    const auto& ends_param = std::get<2>(param);
+    const auto& symbols_param = std::get<3>(param);
+
+    const auto begins = std::make_shared<Constant>(element::i64, indices_shape, begins_param);
+    const auto ends = std::make_shared<Constant>(element::i64, indices_shape, ends_param);
+    const auto symbols = std::make_shared<Constant>(element::u8, Shape{symbols_param.size()}, symbols_param);
```

Comment:
These inputs can have `Paramter::dynamic()`, As static PartialShapes check is not important here.

Reply:
Those inputs are provided as `Constant` ops (constants require static shapes).
So the test should be updated or extended to use Parameters first and then provide Tensor data to accessor if needed.

Reply:
Added more `tensor_accessor` tests for different scenarios.